package.json

```diff
@@ -101,7 +99,7 @@
     "rollup-plugin-commonjs": "^10.0.1",
     "rollup-plugin-dev": "^1.1.3",
     "rollup-plugin-json": "^4.0.0",
-    "rollup-plugin-license": "^2.5.0",
+    "rollup-plugin-license": "^2.6.1",
```

Comment:
Bumps the moment.js transient dependency to 2.29.2 and resolve [CVE-2022-24785](https://nvd.nist.gov/vuln/detail/CVE-2022-24785).